multiaccounts/common/const.go

```diff
@@ -22,3 +22,91 @@ const (
 	CustomizationColorYinYang   CustomizationColor = "yinyang"
 	CustomizationColorBeige     CustomizationColor = "beige"
 )
+
+// ColorToID converts a CustomizationColor to its index.
+func ColorToID(color CustomizationColor) uint32 {
+	switch color {
+	case CustomizationColorPrimary:
+		return 0
+	case CustomizationColorPurple:
+		return 1
+	case CustomizationColorIndigo:
+		return 2
+	case CustomizationColorTurquoise:
+		return 3
+	case CustomizationColorBlue:
+		return 4
+	case CustomizationColorGreen:
+		return 5
+	case CustomizationColorYellow:
+		return 6
+	case CustomizationColorOrange:
+		return 7
+	case CustomizationColorRed:
+		return 8
+	case CustomizationColorFlamingo:
+		return 9
+	case CustomizationColorBrown:
+		return 10
+	case CustomizationColorSky:
+		return 11
+	case CustomizationColorArmy:
+		return 12
+	case CustomizationColorMagenta:
+		return 13
+	case CustomizationColorCopper:
+		return 14
+	case CustomizationColorCamel:
+		return 15
+	case CustomizationColorYinYang:
+		return 16
+	case CustomizationColorBeige:
+		return 17
+	default:
+		return 0
```

Comment:
This might result in redundant code, and both mobile and [desktop](https://github.com/status-im/status-desktop/blob/6cbdcfdcf296ef01419e712e333404c84a754a5e/ui/imports/utils/Utils.qml#L821) clients will need to re-implement this fallback logic.

I've added two extra `ColorToIDFallbackToBlue` and `IDToColorFallbackToBlue` functions for this.